xpdacq/beamtime.py

```diff
@@ -365,17 +365,8 @@ def __init__(self,sample, scanplan):
         self.type = 'sc'
         self.sp = scanplan
         self.sa = sample
-        self.md = self.sp.md
+        self.md = dict(self.sp.md) # create a new dict copy.
```

Comment:
I see.  Perhaps this is a better way.  I was afraid that the metadata was duplicated (items will be both in ``scan.md`` and in ``scan.sp.md`` for example, but I guess it is ok if we only pass ``scan.md`` to the run engine, which is I guess what happens, and then scan kind of dies and doesn't get yaml'd or anything.  Then the ``md`` is not duplicated in the exposure header.